src/MSBuild.Conversion.Project/ProjectRootElementExtensionsForConversion.cs

```diff
@@ -65,6 +65,21 @@ public static IProjectRootElement ChangeImportsAndAddSdkAttribute(this IProjectR
             return projectRootElement;
         }
 
+        public static IProjectRootElement AddCsWinRTReferenceAndComponentProperty(this IProjectRootElement projectRootElement, BaselineProject baselineProject)
+        {
+            if (baselineProject.OutputType == ProjectOutputType.WinMdObj)
+            {
+                var topLevelPropGroup = MSBuildHelpers.GetOrCreateTopLevelPropertyGroup(baselineProject, projectRootElement);
+                topLevelPropGroup.AddProperty(MSBuildFacts.CsWinRTComponentName, "true");
+
+                var itemGroup = projectRootElement.ItemGroups.Where(ig => ig.Items.Where(i => i.ItemType == MSBuildFacts.MSBuildPackageReferenceName).Any())
```

Comment:
Consider renaming to packageReferenceItemGroup.  And I think instead of `ig.Items.Where`, you can do `ig.Items.Any`.

Reply:
;)
```suggestion
                var packageReferenceItemGroup = projectRootElement.ItemGroups.Where(ig => ig.Items.Any(i => i.ItemType == MSBuildFacts.MSBuildPackageReferenceName))
```

Reply:
Thanks for this. I committed this but realized I forgot to update the reference below. Fixed now :)